package.json

```diff
@@ -33,6 +33,7 @@
     "dev": "gatsby develop",
     "build": "gatsby build",
     "heroku-postbuild": "gatsby build",
-    "serve": "gatsby serve"
+    "serve": "gatsby serve",
+    "setup": "npm install --no-save contentful-import yargs-parser axios chalk && node ./setup.js"
```

Comment:
If npm i fails, whole setup fails. I’d decouple this by making these packages dev dependencies so that they are installed during the npm install step and if that fails we can isolate errors to the install step. Here, if things fail it can either be an install error or setup failure and I’d suggest not adding install noise here.

Reply:
Nope we can not do this hence this would extend the already long install time even further for the CLI guide.

Reply:
@zeusdeux yeah, the reason why these packages are here because we don't want to increase installing dependencies time for the CLI guide.